File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/PictureBox/PictureBoxPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Concepts.ClientAPI.PictureBox
{
    public partial class PictureBoxPage : UserControl
    {
        public PictureBoxPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the SelectedIndexChanged event of the mobjListBox control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjListBox_SelectedIndexChanged(object sender, EventArgs e)
        {
            VWGClientContext.Current.Invoke("onSelectedChanged");
        }

    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/PictureBox/PictureBoxPage.vb
Namespace CompanionKit.Concepts.ClientAPI.PictureBox

	Public Class PictureBoxPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        ''' <summary>
        ''' Handles the SelectedIndexChanged event of the mobjListBox control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjListBox_SelectedIndexChanged(sender As System.Object, e As System.EventArgs) Handles mobjListBox.SelectedIndexChanged
            VWGClientContext.Current.Invoke("onSelectedChanged")
        End Sub
    End Class

End Namespace